{
  "modified": "2025-08-28T07:11:18Z",
  "published": "2025-08-28T07:11:18Z",
  "schema_version": "1.5.0",
  "id": "",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "libfwupdplugin1"
      },
      "versions": [
        "2.0.0",
        "1.0.0"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "95c430f5929b13c8ca7499575421ed335064589190cddeb4c70b64e63f819365",
        "import_time": "2025-08-29T06:42:44.307900166Z",
        "id": "RLUA-2025-04188",
        "modified_time": "2025-08-28T07:11:18Z",
        "versions": [
          "2.0.0",
          "1.0.0"
        ]
      }
    ]
  }
}
